NetKAN/KashCorpSA.netkan

```diff
@@ -0,0 +1,20 @@
+{
+        "spec_version" : "v1.4",
+        "identifier" : "KashCorpSA",
+        "name" : "KashCorp Shared Assets (Virtual mod)",
```

Comment:
You don't need to specify a mod as virtual, a not very small part of mods in the repository are in one way or another. This distinction could be confusing to users.